db/migrations/versions/034_nullable_reporting_round.py

```diff
@@ -0,0 +1,26 @@
+"""nullable reporting_round
+
+Revision ID: 034_nullable_reporting_round
+Revises: 033_project_geospatial_data
+Create Date: 2024-05-14 09:01:17.945873
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "034_nullable_reporting_round"
+down_revision = "033_project_geospatial_data"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("submission_dim", schema=None) as batch_op:
```

Comment:
Hello again @samuelhwilliams just checking then, we are making this column nullable rather than dropping, or is there gonna be another PR for that?

Reply:
We will drop it in the next PR :)